modules/javafx.graphics/src/main/java/javafx/stage/StageStyle.java

```diff
@@ -32,12 +32,16 @@
 public enum StageStyle {
 
     /**
-     * Defines a normal {@code Stage} style with a solid white background and platform decorations.
+     * Defines a normal {@code Stage} style with a solid background and platform decorations.
      */
     DECORATED,
 
     /**
-     * Defines a {@code Stage} style with a solid white background and no decorations.
+     * Defines a {@code Stage} style with no window decorations, such as a title bar,
+     * borders, or window controls.
+     * This style allows window operations such as resize, minimize, maximize
+     * and fullscreen to be either programmatically controlled or achieved through
+     * platform-specific functions, such as key shortcuts or menu options.
```

Comment:
It looks like you intend the 2nd part to be a new paragraph:
```suggestion
     * Defines a {@code Stage} style with no window decorations, such as a title bar,
     * borders, or window controls.
     * <p>
     * This style allows window operations such as resize, minimize, maximize
     * and fullscreen to be either programmatically controlled or achieved through
     * platform-specific functions, such as key shortcuts or menu options.
```

Reply:
+1